### ISSUE TITLE
No 'data driven sample scenario' generated in the Serenity Html Report

### ISSUE THREAD

**Issue by ereticul** [serenity-bdd/serenity-cucumber#10]
Hi, 
Perhaps it could more related to Serenity-Core, it actually could be also an issue reproduced for Serenity-JBehave. 

When I mean sample scenario, there is a section above of Examples table (e.g 'http://wakaleo.com/thucydides-sample-reports/c2c1decb61b963a0194169cd710f500f63ec570e622f9b9bd8b7870f21ed3c9c.html')

Steps to reproduce: 
(1) Execute the scenario below:
Scenario: Non-web Data-driven testing from an external CSV file

Given the data in data/names-data.csv
When we enter this data
Then the values should be correct

The output is as follow, the 'Examples' table is populated:
Examples:
FIRSTNAME	LASTNAME	EXPECTEDFIRSTNAME	EXPECTEDLASTNAME
Joe	                Smith	        Will	                                        Smith
Forrest	          Bill	                Scott	                                George

But the Scenario Sample section is blank. 

TestOutcome.java -> there is a check first step to have children, that's why it has been failed to get the sample:
public String getDataDrivenSampleScenario() {
        if(this.isDataDriven() && !this.getTestSteps().isEmpty() && ((TestStep)this.getTestSteps().get(0)).hasChildren()) {
            TestStep firstExample = (TestStep)this.getTestSteps().get(0);
            StringBuilder sampleScenario = new StringBuilder();

Maybe this is a cosmetic issue, that's not really works like a Scenario Outline with Example data table. What do you think?

Thanks,
Andy

**Comment by wakaleo** [comment]
Yes, this should be in core. It is an enhancement because the Scenario Outline support is specific to Cucumber (there is an equivalent in JBehave), and the equivalent support needs to be added to the JUnit driver.

**Comment by wakaleo** [comment]
Could you provide a sample project to reproduce this?
